tlatools/org.lamport.tlatools/src/tla2sany/explorer/Explorer.java

```diff
@@ -96,12 +97,27 @@ private boolean getLine() {
 		return true;
 	}
 
+	/**
+	 * Print diagnostics about the given module that require access to the
+	 * semantic error log.
+	 *
+	 * @param node The module to print info about.
+	 */
+	private void printModuleNodeDetails(ModuleNode node) {
+		System.out.println("Constant module: " + node.isConstant(this.errors));
```

Comment:
This uses `System.out`, whereas the code base generally uses `ToolIO.out`.  IIRC, the latter works with the Toolbox.